.dockerfiles/latest/x86-64-pc-windows-msvc/build-and-push.ps1

```diff
@@ -0,0 +1,7 @@
+# You should already be logged in to DockerHub when you run this.
+$ErrorActionPreference = 'Stop'
+
+$dockerfileDir = Split-Path $script:MyInvocation.MyCommand.Path
+
+docker build --pull -t "ponylang/ponyc:windows" $dockerfileDir
```

Comment:
We don't do --pull for any others. I don't think we need unless it's windows specific.

Reply:
Not sure what you mean... in master it looks like all of the `build-and-push.bash` scripts under `.dockerfiles` use `--pull`, so I copied that.